types/pubsub-js/index.d.ts

```diff
@@ -19,6 +19,7 @@ declare namespace PubSubJS {
 
     interface Subscribe{
         subscribe(message: any, func: Function): any;
+        subscribeOnce(message: any, func: Function): any;
```

Comment:
As discussed in [the README](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes), we don't recommend using the type `Function`. Can you be more specific, even if it's `(...args: any[]) => any'? Thanks!